### DISCUSSION TITLE
Create additional Worktrees for a Repository

### DISCUSSION THREAD

**Discussion by rushsteve1** [go-git/go-git#885] [Q&A] [answered]
Hello! I'm trying to figure out how to create multiple Worktrees for a given Repository, similar to what is described in [the Git CLI docs here](https://git-scm.com/docs/git-worktree). The `Repository.Worktree()` function just returns the primary Worktree, or throws an error if it is a Bare Repository.

My exact use-case is I want to create a Bare Repository on-disk, and then create a number of Worktrees in-memory that I can edit.

**Comment by makkes** [comment, marked as answer]
The [compatibility doc says](https://github.com/go-git/go-git/blob/master/COMPATIBILITY.md#other-features) "Multiple worktrees are not supported."
